lib/widgets/action_sheet.dart

```diff
@@ -247,30 +248,56 @@ class ReactionButtons extends StatelessWidget {
-      child: Row(
-        mainAxisAlignment: MainAxisAlignment.spaceAround,
-        children: List.unmodifiable(popularEmojis.map((emoji) {
-          final selfVoted = hasSelfVote(emoji);
-          return IconButton(
-            onPressed: () => _onPressed(emoji, selfVoted, zulipLocalizations),
-            isSelected: selfVoted,
-            style: IconButton.styleFrom(
-              padding: EdgeInsets.zero,
-              splashFactory: NoSplash.splashFactory,
-              shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(3.5)),
-              tapTargetSize: MaterialTapTargetSize.shrinkWrap,
-              visualDensity: VisualDensity.compact,
-            ).copyWith(backgroundColor: WidgetStateColor.resolveWith((states) =>
-              states.any((e) => e == WidgetState.pressed || e == WidgetState.selected)
-                ? designVariables.contextMenuItemBg.withFadedAlpha(0.20)
-                : Colors.transparent)),
-            icon: UnicodeEmojiWidget(
-              emojiDisplay: emoji.emojiDisplay as UnicodeEmojiDisplay,
-              notoColorEmojiTextSize: 20.1,
-              size: 24));
-        })))
+      child: Row(children: [
+        Flexible(child: Padding(
+          padding: const EdgeInsets.symmetric(vertical: 8),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: List.unmodifiable(popularEmojis.map((emoji) {
+              final selfVoted = hasSelfVote(emoji);
+              return IconButton(
+                onPressed: () => _onPressed(emoji, selfVoted, zulipLocalizations),
+                isSelected: selfVoted,
+                style: IconButton.styleFrom(
+                  padding: EdgeInsets.zero,
+                  splashFactory: NoSplash.splashFactory,
+                  shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(3.5)),
+                  tapTargetSize: MaterialTapTargetSize.shrinkWrap,
+                  visualDensity: VisualDensity.compact,
+                ).copyWith(backgroundColor: WidgetStateColor.resolveWith((states) =>
+                  states.any((e) => e == WidgetState.pressed || e == WidgetState.selected)
+                    ? designVariables.contextMenuItemBg.withFadedAlpha(0.20)
+                    : Colors.transparent)),
+                icon: UnicodeEmojiWidget(
+                  emojiDisplay: emoji.emojiDisplay as UnicodeEmojiDisplay,
+                  notoColorEmojiTextSize: 20.1,
+                  size: 24));
+            }))))),
+        TextButton(
+          onPressed: () {
+            showEmojiPickerSheet(pageContext: pageContext, message: message);
+          },
+          style: TextButton.styleFrom(
+            padding: const EdgeInsets.fromLTRB(12, 12, 4, 12),
+            splashFactory: NoSplash.splashFactory,
+            foregroundColor: designVariables.contextMenuItemText,
+            shape: const RoundedRectangleBorder(borderRadius: BorderRadius.zero)
```

Comment:
Why `RoundedRectangleBorder` where the rectangle is not rounded (border radius is zero)? 🙂 (Here and in another place in the PR.)